src/components/useraccount/MyOrders.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <div class="col-sm-9">
+    <div v-if="me"
+         class="my-orders col-sm-12">
+      <div class="my-orders-content">
+        <div class="row">
+          <div class="col-sm-12">
+            <div class="my-orders-title hidden-xs">
+              <h4>{{ $t('myOrders')}}</h4>
+            </div>
+          </div>
+        </div>
+        <div class="my-orders-table-wrapper">
+          <div class="row">
+            <div class="col-sm-2 col-xs-4">
+              <div class="my-orders-table-titles">
+                <h5>{{ $t('orderNumber')}}</h5>
+              </div>
+            </div>
+            <div class="col-sm-2 hidden-xs">
+              <div class="my-orders-table-titles">
+                <h5>{{ $t('date')}}</h5>
+              </div>
+            </div>
+            <div class="col-sm-2 col-xs-4">
+              <div class="my-orders-table-titles">
+                <h5>{{ $t('total')}}</h5>
+              </div>
+            </div>
+            <div class="col-sm-2 hidden-xs">
+              <div class="my-orders-table-titles">
+                <h5>{{ $t('paymentStatus')}}</h5>
+              </div>
+            </div>
+            <div class="col-sm-2 hidden-xs">
+              <div class="my-orders-table-titles">
+                <h5>{{ $t('shipmentStatus')}}</h5>
+              </div>
+            </div>
+            <div class="col-sm-2 col-xs-4">
+              <div class="my-orders-table-titles">
+                &nbsp;
+              </div>
+            </div>
+          </div>
+        </div>
+        <div v-for="order in me.orders.results"
+             :key="order.id"
+             data-test="order-list"
+             class="my-orders-table-order">
+          <div class="row">
+            <div class="col-sm-2 col-xs-4" >
+              <span data-test="order-number">
+                {{ order.orderNumber || "-" }}
+                </span>
+            </div>
+            <div data-test="order-date"
+                 class="col-sm-2 hidden-xs">
+              <BaseDate :dateObj="{ date: order.createdAt, format: 'short' }"/>
```

Comment:
It is a bit unnatural to pass an object as props in Vue when you can pass them separately.
`<BaseDate :date="order.createdAt" :format="short"/>`